cassandra_migrate/cli.py

```diff
@@ -70,6 +70,8 @@ def main():
                         migrations are run against. This option must be used in
                         conjuction with the -k option. This option is ignored
                         unless the -s option is provided.""")
+    parser.add_argument('-v', '--protocol-version', type=int, default=None,
```

Comment:
Can we add a `choices` argument here matching the enumeration in `ProtocolVersion.SUPPORTED_VERSIONS` ([docs](https://datastax.github.io/python-driver/api/cassandra.html#cassandra.ProtocolVersion))

Reply:
OK, I will hardcode versions 1, 2, 3, 4, 5 instead of passing `ProtocolVersion.SUPPORTED_VERSIONS` to choices, to avoid potential import errors and to keep it separate from how ProtocolVersion might work internally in the future